python/lsst/sims/maf/plots/plotHandler.py

```diff
@@ -566,10 +566,10 @@ def saveFig(self, fignum, outfileRoot, plotType, metricName, slicerName,
         fig = plt.figure(fignum)
         plotFile = outfileRoot + '_' + plotType + '.' + self.figformat
         if self.trimWhitespace:
-            fig.savefig(os.path.join(self.outDir, plotFile), figformat=self.figformat, dpi=self.dpi,
+            fig.savefig(os.path.join(self.outDir, plotFile), dpi=self.dpi,
                         bbox_inches='tight')
         else:
-            fig.savefig(os.path.join(self.outDir, plotFile), figformat=self.figformat, dpi=self.dpi)
+            fig.savefig(os.path.join(self.outDir, plotFile), dpi=self.dpi)
```

Comment:
Again, please put format choice back in, using proper keyword argument (format = self.figformat).